spec/rspec/matchers/expecteds_for_multiple_diffs_spec.rb

```diff
@@ -0,0 +1,82 @@
+module RSpec
+  module Matchers
+    RSpec.describe ExpectedsForMultipleDiffs do
+
+      # Setup fake differ
+      let(:differ_factory) { double("Differ factory method", :call => differ) }
+      let(:differ) { double("Differ") }
+
+      before do
+        allow(differ).to receive(:diff) do |_actual, expected|
+          [_actual, expected].inspect
+        end
+      end
+
+      let(:message) { "a message" }
+      let(:actual) { "actual value" }
+
+      let(:wrapped_value) { described_class.from("expected value") }
+
+      let(:matcher_1) { double("Matcher", :description => "matcher 1 description", :expected => "expected 1") }
+      let(:matcher_2) { double("Matcher", :description => "matcher 2 description", :expected => "expected 2") }
+      let(:matcher_3) { double("Matcher", :description => "matcher 3 description", :expected => "expected 3") }
```

Comment:
I'd prefer to see `instance_double` used (maybe `instance_double(BaseMatcher)`) as that's safer.